src/promptflow/promptflow/azure/operations/_flow_operations.py

```diff
@@ -20,26 +23,40 @@
 from azure.core.exceptions import HttpResponseError
 
 from promptflow._sdk._constants import (
+    CLIENT_FLOW_TYPE_2_SERVICE_FLOW_TYPE,
+    DAG_FILE_NAME,
     FLOW_TOOLS_JSON,
     LOGGER_NAME,
     PROMPT_FLOW_DIR_NAME,
     WORKSPACE_LINKED_DATASTORE_NAME,
+    FlowType,
 )
+from promptflow._sdk._errors import FlowOperationError
+from promptflow._sdk._logger_factory import LoggerFactory
 from promptflow._sdk._utils import PromptflowIgnoreFile, generate_flow_tools_json
 from promptflow._sdk._vendor._asset_utils import traverse_directory
-from promptflow.azure._entities._flow import Flow
+from promptflow._telemetry.activity import ActivityType, monitor_operation
+from promptflow.azure._constants._flow import DEFAULT_STORAGE
+from promptflow.azure._entities._flow import Flow as AzureFlow
+from promptflow.azure._load_functions import load_flow as load_azure_flow
```

Comment:
why need to rename this?

Reply:
no need to differentiate Flow AzureFlow?

Reply:
Updated, but may need to discuss what Flow the create method returns, Flow or AzureFlow